### PR TITLE
Fixing issue with "value-keyword-case" stylelint rule

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "extends": ["stylelint-config-recommended-scss", "stylelint-config-standard"],
   "rules": {  
     "at-rule-no-unknown": null,
-    "scss/at-rule-no-unknown": true
+    "scss/at-rule-no-unknown": true,
+    "value-keyword-case": null
```
Comment (probil) [inline .stylelintrc]: why disabling is necessary?

**Comment by liubich** [comment]
Looks like it works inproperly.
It raise error on font name "Roboto", suggests to make it all lowercase.
Error exists at CircleCI only and I can't reproduce it on my machine.

**Comment by probil** [comment]
@liubich Could you send me screenshot or link where I can check it?

**Comment by liubich** [comment]
Screenshot from CircleCI job:

image

**Comment by probil** [comment]
It seems like there is an issue for that 
https://github.com/stylelint/stylelint/issues/1101

**Comment by liubich** [comment]
I fixed this issue in more flexible way by adding "ignoreKeywords" option to this rule.

**Pull request merged**